src/fix/main.c

```diff
@@ -325,7 +325,7 @@ do { \
 				tryReadSlice("MA5");
 				mbc = BANDAI_TAMA5;
 				break;
-			case 'P':
+			case 'P': {
```

Comment:
A variable is declared inside this `switch` case, so in C++ needs its own scope.